### ISSUE TITLE
Incompatible with Odoo 9!

### ISSUE THREAD

**Issue by Nolaan** [Odoo-mobile/crm#58]
Cannot log into self managed host with the up to date version of Odoo 9.

**Comment by dpr-odoo** [comment]
@Nolaan 

Yes, It is not compatible with Odoo 9.0 yet. Will soon available for Odoo 9.0
